packages/aggregator/src/__tests__/createAggregator.test.js

```diff
@@ -2,8 +2,6 @@ import { DataBroker } from '@tupaia/data-broker';
 import { Aggregator } from '../Aggregator';
 import { createAggregator } from '../createAggregator';
 
-jest.mock('@tupaia/data-broker');
```

Comment:
I tried removing this locally and I got database connection errors... Which makes sense since `new DataBroker` calls `getModels`, which creates a new DB connection.

By the way, I don't have a `.env` file for `aggregator` locally, but it shouldn't be required.

Reply:
That's weird. I don't have `.env` file for aggregator as well but test pass without this mock. I guess it uses the `.env` file in `database` directly.

Reply:
I don't think so, I also have a .`env` in my database packages... 🤷‍♀️